### ISSUE TITLE
Add a nice sharable link to report issues to JIRA

### ISSUE THREAD

**Issue by mgifford** [mgifford/vital-core#133]
We should be able to report per issue accessibility errors like we can in open-scans.

This would make it easier for us to capture the issues in our project management tools
